src/core/network/polling.ts

```diff
@@ -15,9 +15,11 @@ export class Polling {
   private readonly abortController = new AbortController()
   private skipOffsetSync = false
   private offset = 0
+  public limit: number | undefined
   constructor(
     private readonly telegram: ApiClient,
-    private readonly allowedUpdates: readonly tt.UpdateType[]
+    private readonly allowedUpdates: readonly tt.UpdateType[],
+    private readonly timeout: number
```

Comment:
I said 

> It might make sense to make it configurable in the future. (https://github.com/telegraf/telegraf/discussions/1234#discussioncomment-215009)

Not "It makes sense right now"

Reply:
This is the result of a merge. #1247 could already do it a few weeks ago. I can of course remove it again and reintroduce it in a subsequent PR.

> Not "It makes sense right now"

Do you mean "it makes no sense right now" so we must remove it?

Reply:
Yes. Except I see no point in reintroducing it right now.